mne/utils.py

```diff
@@ -899,6 +901,63 @@ def _update_doc(self, olddoc):
         return newdoc
 
 
+def deprecate_parameter_rename(deprecated_in, removed_in,
+                               old_param, new_param,
+                               current_version=__version__,
+                               details=None,
+                               transform=None):
+    """Mark a function parameter as deprecated.
+
+    Issue a warning when the function is called with the `old_parameter`.
+
+
+    Parameters
+    ----------
+    extra: string
+    deprecated_in: string
+        When was deprecated.
+    removed_in: string
+        When `old_param` would no longer be accepted.
+    old_param: string
+        Old parameter name.
+    new_param: string
+        New parameter name.
+    current_version: string
+        MNE-python current version. This parameter is meant for test purposes.
+    details: string
+        Extra information to append to the warning message.
+    transform: callable
+        optional. This is meant to transform a valid old parameter into a valid
+        new parameter.
+    """
+    _MSG_KEYS = {'old_param': old_param,
+                 'new_param': new_param,
+                 'deprecated_in': deprecated_in,
+                 'removed_in': removed_in,
+                 'old_param': old_param,
+                 'new_param': new_param,
+                 'details': details}
+    _MSG = ('`{old_param}` is deprecated and will be replaced by `{new_param}`'
+            ' in {deprecated_in}. `{old_param}` will be no longer present'
+            ' starting in {removed_in}. {details}').format(**_MSG_KEYS)
+
+    # XXX: __version__ should be tested
+
+    def true_decorator(f):
+        @wraps(f)
```

Comment:
`wraps` is not be enough to get the docstring correct here. You need to use the `externals/decorator` functionality.